resource/out

```diff
@@ -40,38 +41,37 @@ def create_client(src_dir, source={}, params={}):
 
     data = {}
     with open('/tmp/build/put/webapp-source/deploy.json') as input:
-        print('Parsing deploy.json')
-        data = json.load(input)
+        try:
```

Comment:
Maybe worth keeping the mention of the beginning of the parsing of the `deploy.json`.
If anything would give the users some peace of mind that this is being done as this file is directly edited by them. 

Unless users don't really care about this unless it fails/breaks.
I suppose we can always add back easily.

Reply:
:+1: